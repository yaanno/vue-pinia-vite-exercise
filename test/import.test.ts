describe('import vue components', () => {
  test('normal imports as expected', async () => {
    const cmp = await import('../src/App.vue')
    expect(cmp).toBeDefined()
    expect(cmp).toMatchSnapshot()
  })
})
