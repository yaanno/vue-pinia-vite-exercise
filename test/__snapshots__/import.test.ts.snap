// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`import vue components > normal imports as expected 1`] = `
{
  "default": {
    "__file": "/Users/janoshardi/Workspace/vue-exercise-app/src/App.vue",
    "__name": "App",
    "render": [Function],
    "setup": [Function],
  },
}
`;
